app/css/Evaluation.css

```diff
@@ -25,11 +25,9 @@ body {
 .btn {
   margin-top: 10px;
   padding: 10px 20px;
-  font-size: 16px;
```

Comment:
change the 16px with 1rem as it is the same by default and is better for scalability